backend/app/controllers/spree/admin/users_controller.rb

```diff
@@ -35,6 +35,10 @@ def create
       end
 
       def update
+        if params[:user][:password].blank? && params[:user][:password_confirmation].blank?
+          params[:user].delete(:password)
+          params[:user].delete(:password_confirmation)
+        end
```

Comment:
Rather than deleting attributes from `params` I think this would be better done by updating the `user_params` function.